### ISSUE TITLE
Activities ranked/adjusted by risk

### ISSUE THREAD

**Issue by nathanawmk** [microCOVID/microCOVID#1186]
**Is your feature request related to a situation you're trying to understand with microCOVID? Please describe.**
A clear and concise description of the situation you're trying to understand by using microCOVID.

Yes. For example I am planning to do 10 activities on the weekends and I want to be able to key in these 10 activities, hit the submit button, and see a response of these risk activities rank - this will guide me in my decision

**Describe the solution you'd like**
As described above

**Additional context**
I like to extend on this project for a more specified context for Singapore - can I do so? 

**Relevant Research or Citations**
If you have any data or studies that back this feature, please link them here. (If applicable)

**Comment by blanchardjeremy** [comment]
@nathanawmk  - Are you looking to see the 10 activities side by side? This is exactly what the Risk Tracker is designed for: https://microcovid.org/tracker

It's not something we intend to build into the calculator.

**Comment by blanchardjeremy** [comment]
Yes, it is an open-source project. You can do anything in line with the license, including forking and modifying any of the code and repo contents: https://github.com/microCOVID/microCOVID/blob/main/LICENSE
